docs/reference/ml/df-analytics/apis/put-inference.asciidoc

```diff
@@ -94,6 +94,14 @@ The field name to encode.
 `frequency_map`::
 (Required, object map of string:double)
 Object that maps the field value to the frequency encoded value.
+
+`custom`::
+(Optional, boolean)
+Boolean value indicating if the analytics job created the preprocessor
+or if a user provided it. This adjusts the feature importance calculation.
+When `true`, the feature importance calculation returns importance for the
+processed feature. When `false`, the total importance of the original field
+is returned. Default is `false`.
```

Comment:
@elastic/ml-docs what do y'all think?

Reply:
@benwtrent The text looks good to me. As you use it in multiple places, maybe it would be good to define `custom` in the ml-shared.asciidoc and reference it from there. Please let me know if I can help.